js/success-message.js

```diff
@@ -0,0 +1,29 @@
+import {isEscapeKey} from './util.js';
+
+const SHOW_TIME = 3000;
+
+const successMessageContainer = document.querySelector('#success').content.querySelector('.success');
+const successMessage = successMessageContainer.cloneNode(true);
+
+const hideSuccessMessage = () => {
+  successMessage.style.display = 'none';
+};
+
+const onSuccessMessageEscKeydown = (evt) => {
+  if (isEscapeKey) {
```

Comment:
Тут должен быть вызов функции
```suggestion
  if (isEscapeKey()) {
```